helpers/decodeHexString.ts

```diff
@@ -10,6 +10,7 @@ export function decodeHexString(hexString: string) {
     if (e instanceof Error) {
       throw new Error(`Invalid hex string: ${e.message}`);
     } else {
```

Comment:
I don't see any other way than to ignore the rule here — we _want_ to attempt to stringify the error even if it's of type `unknown`.